packages/manager/modules/pci/src/projects/project/public-gateways/public-gateways.html

```diff
@@ -12,4 +12,100 @@
             </oui-guide-menu-item>
         </oui-guide-menu>
     </oui-header>
+    <div class="row">
+        <p
+            class="col-md-8"
+            data-translate="pci_projects_project_public_gateways_intro"
+        ></p>
+    </div>
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+    <oui-datagrid
+        data-rows="$ctrl.gateways"
+        data-ng-if="$ctrl.gateways.length > 0"
+    >
+        <oui-datagrid-topbar>
+            <button
+                class="oui-button oui-button_secondary oui-button_icon-left mb-2"
+                data-ng-click="$ctrl.onAddPublicGatewayClick()"
+                type="button"
+                data-track-on="click"
+                data-track-type="navigation"
```

Comment:
**question:** Since the tracking is implemented in the callback `onAddPublicGatewayClick`. Is this required?

Reply:
Removed as not required.